pkg/handler/collector/deps_dev/deps_dev_test.go

```diff
@@ -262,6 +263,15 @@ func Test_depsCollector_RetrieveArtifacts(t *testing.T) {
 				t.Errorf("Wanted %v elements, but got %v", len(tt.want), len(collectedDocs))
 			}
 			for i := range collectedDocs {
+				// Why do we do this here instead of in the test case definition?
```

Comment:
`normalizeTimeStampAndScorecard` is done such that the compare passes for the test (as the timestamps maybe different)

Reply:
Not sure if I answered your question. Let me know and we can discuss.

Reply:
Fixed in e114cf3e688203b1eecd3c300cce9983b940c605. I actually think we should use a better comparison mechanism like `github.com/stretcher/testify/assert` but that is a discussion for another day.